src/AppBundle/Resources/views/Person/show.json.twig

```diff
@@ -0,0 +1,62 @@
+
+   {
+        
+   "@context": "http://schema.org",
+   "@type": "Person",
+
+   "url": "{{ url('person_show', {'id': person.id}) }}",
+   "familyName": "{{ person.firstName }}",
+   "givenName": "{{ person.lastName }}",
+   {% if person.title %}
+   "jobTitle": "{{ person.title }}",
+   {% endif %}
+   "birthDate": "{{ person.dob }}",
+   "deathDate": "{{ person.dod }}",
+   {% if person.gender == 'F' %}
+   "gender": "female",         
+   {% elseif person.gender == 'M' %}
+   "gender": "male",        
+   {% else %}
+   "gender": "unknown", 
+   {% endif %}
+       
+   "birthPlace": {
+   {% if person.cityOfBirth and person.cityOfBirth.geonameid != 0 %}
```

Comment:
So, if person.cityOfBirth is null, this will look like 

`"birthPlace": {}`

Will that be a problem? Seems like it would be better to not include anything in this case.